flink-python/pyflink/datastream/stream_execution_environment.py

```diff
@@ -553,6 +553,45 @@ def set_python_executable(self, python_exec: str):
             .getEnvironmentConfig(self._j_stream_execution_environment)
         env_config.setString(jvm.PythonOptions.PYTHON_EXECUTABLE.key(), python_exec)
 
+    def add_jars(self, *jars_path: List[str]):
```

Comment:
I think the signature should be:
```suggestion
    def add_jars(self, *jars_path: str):
```

What do you think?